### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by Vrushti24**

### CODE REVIEW

**Comment by ghost** [comment]
@Vrushti24 what if we make a react component?

**Comment by Vrushti24** [comment]
Means ? This doesn't look good?

**Comment by ghost** [comment]
Did you test it? Please create a branch and push I'll test locally

**Comment by Vrushti24** [comment]
Yeah I have test it already sent the video of that and yes branch was also
created

**Comment by Vrushti24** [comment]
Sir any updates??

**Comment by ghost** [comment]
@vrushti I have merged. Can you share the snapshot?

**Comment by Vrushti24** [comment]
It's a video if I share video than it's okay ??

**Comment by ghost** [comment]
Yes. Fine. Share on discord channel as well.

**Comment by Vrushti24** [comment]
Will do, thanks!

File: public/index.html [hunk 1]
```diff
@@ -25,10 +19,27 @@
...
+
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body class="welcome">
+
+    <span id="splash-overlay" class="splash"></span>
+
+    <span id="welcome" class="z-depth-4">
+      <img src = 'E:\Open Source\idea-lab\public\logo192.png' width="50" height="50"></img>
```
Comment (sanjay270899) [inline public/index.html:31]: @Vrushti24 You have used the absolute path for the image.

File: public/index.html [hunk 2]
```diff
@@ -25,10 +19,27 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
```
Comment (sanjay270899) [inline public/index.html:31]: I think by mistake you have removed `root` div container.

**Comment by Vrushti24** [comment]
Yeah actually I apologised for that on discord it was by mistake

**Comment by sanjay270899** [comment]
@Vrushti24 I think, it will be good if you send a PR for fixing this.

**Comment by Vrushti24** [comment]
Is it necessary I mean already I said on discord about that so I thought
problem was solved

**Pull request merged**